packages/vite-node/src/cli.ts

```diff
@@ -48,11 +49,12 @@ Usage:
   $ vite-node [options] [files]
 
 Options:
-  -r, --root <path>      ${dim('[string]')} use specified root directory
-  -c, --config <file>    ${dim('[string]')} use specified config file
-  -w, --watch           ${dim('[boolean]')} restart on file changes, similar to "nodemon"
-  -s, --silent          ${dim('[boolean]')} do not emit errors and logs
-  --vue                 ${dim('[boolean]')} support for importing Vue component
+  -r, --root <path>           ${dim('[string]')} use specified root directory
+  -c, --config <file>         ${dim('[string]')} use specified config file
+  -w, --watch                ${dim('[boolean]')} restart on file changes, similar to "nodemon"
+  -s, --silent               ${dim('[boolean]')} do not emit errors and logs
+  --vue                      ${dim('[boolean]')} support for importing Vue component
+  -i, --inline-deps <deps>    ${dim('[string]')} inline specified dependencies
```

Comment:
I'm not convinced that this is the best way to show that this flag may be set several times to inline several dependencies. Does anybody have a better idea?

Reply:
I think we use `[...files]` for vitest.